libs/datasets/timeseries.py

```diff
@@ -853,25 +853,23 @@ def read_from_pointer(pointer: dataset_pointer.DatasetPointer) -> "MultiRegionDa
         wide_dates_df.columns = pd.to_datetime(wide_dates_df.columns)
         wide_dates_df = wide_dates_df.rename_axis(columns=CommonFields.DATE)
 
-        static_df = pd.read_csv(
-            pointer.path_static(), dtype={CommonFields.FIPS: str}, low_memory=False
-        )
-
-        # TODO(tom): Delete this once annotation file is retired
-        if pointer.path_annotation().is_file():
-            annotations = pd.read_csv(pointer.path_annotation(), low_memory=False)
-            tag_df_to_concat.append(annotations)
+        tag_df = pd.concat(tag_df_to_concat)
+        if not bucketed:
+            tag_df_add_all_bucket_in_place(tag_df)
 
-        dataset = MultiRegionDataset.from_timeseries_wide_dates_df(
+        return MultiRegionDataset.from_timeseries_wide_dates_df(
             wide_dates_df, bucketed=bucketed
-        ).add_static_values(static_df)
-        if tag_df_to_concat:
-            tag_df = pd.concat(tag_df_to_concat)
-            if not bucketed:
-                tag_df_add_all_bucket_in_place(tag_df)
-            dataset = dataset.append_tag_df(tag_df)
+        ).append_tag_df(tag_df)
 
-        return dataset
+    def add_static_csv_file(self, path_or_buf: Union[pathlib.Path, TextIO]) -> "MultiRegionDataset":
+        static_df = pd.read_csv(path_or_buf, dtype={CommonFields.FIPS: str}, low_memory=False)
+        return self.add_static_values(static_df)
+
+    @staticmethod
+    def read_from_pointer(pointer: dataset_pointer.DatasetPointer) -> "MultiRegionDataset":
```

Comment:
Are we keeping these in the meantime?

Reply:
Yes, removing them is more than I want to bite off when trying to debug something else.